src/tool/cppxlang/strings/base_chrono.h

```diff
@@ -17,32 +17,32 @@ namespace xlang
 
 namespace xlang::impl
 {
-    template <> struct abi<Windows::Foundation::TimeSpan>
+    template <> struct abi<Foundation::TimeSpan>
     {
         using type = int64_t;
     };
 
-    template <> struct abi<Windows::Foundation::DateTime>
+    template <> struct abi<Foundation::DateTime>
     {
         using type = int64_t;
     };
 
-    template <> struct name<Windows::Foundation::TimeSpan>
+    template <> struct name<Foundation::TimeSpan>
     {
-        static constexpr auto & value{ u8"Windows.Foundation.TimeSpan" };
+        static constexpr auto & value{ u8"Foundation.TimeSpan" };
```

Comment:
Shouldn't this name be "xlang.Foundation.TimeSpan"?

Reply:
I looked thru the .il file and see that you've removed the xlang namespace across the board. I'm OK with this, though I worry a bit about name collisions. I suspect "Foundation" is a lot more common than "xlang.Foundation"

Reply:
Not yet, it isn't. We're the only ones. :)

But seriously, shouldn't the platform developers have the prerogative to reserve a good namespace for themselves? The C++ standard library has `std`, C++/CX has/had `Platform`.

Reply:
Good point. Works for me.